### ISSUE TITLE
Test ETS with upcoming iPadOS 13. 

### ISSUE THREAD

**Issue by JFBeaudoin** [cadets-ca/ets#34]
We need to test ETS with the Public Beta release of iPadOS 13 to make sure any new features of iPadOS 13 won't interfere with ETS.

We need to especially test how ETS behaves with Slide Over (multitasking of apps) and how the application looks when in the new Dark mode. Chances are the text won't be readable in the iPad is in Dark mode AND the text is in Black. 

Other upcoming task might be opened depending of the outcome of the current testing.

**Comment by kirvanp** [comment]
The app has supported the various multi-tasking modes for many years, so I don't expect any trouble there. 

The app does not have any Dark Mode support. Seeing as the Air Cadet Gliding Program only operates in day VFR, I wouldn't put a lot of priority into it. Apps that don't support it remain black on white in iOS 13 which is a bit jarring to the user if their other apps have switched to white on black, but they remain readable. It seems that SwiftUI supports Dark Mode by default, so over time as portions of the app get added or rewritten in SwiftUI it should come along.

**Comment by JFBeaudoin** [comment]
Ok then! Thank you for replying that fast! 

Then, we should maybe just test and know what is going on... 

Even though we might not prioritize task related to iPadOS 13 improvements, we need to be aware of all the behaviors so the Service Desks are aware if users call-in for support.

**Comment by kirvanp** [comment]
Yup, definitely should be tested on it. There are several things in iOS 13 that are potentially going to allow us to simplify the app and make it more reliable:

1) New methods for syncing Core Data content over iCloud which should save thousands of lines of boilerplate code
2) SwiftUI allows the interface to essentially update itself whenever the data in the app changes, rather than having to do that manually 
3) There are new methods for TableViews that allow them to automatically rearrange themselves when the data source changes rather than the programmer having to figure out which row moved where and so on

Overall I'm pretty excited about it. The WWDC 2019 videos are well worth watching.

**Comment by JFBeaudoin** [comment]
Will do! 

I've been listening to the WWDC 2019 keynote but haven't had a look at the detailed videos of the features.

**Comment by kirvanp** [comment]
Well I might recommend [Introducing Swift UI](https://developer.apple.com/videos/play/wwdc2019/204/) [Using Core Data with CloudKit](https://developer.apple.com/videos/play/wwdc2019/202/) and [Advances in UI Data Sources](https://developer.apple.com/videos/play/wwdc2019/220/) as the most inspiring talks I've watched so far.
